vochain/census.go

```diff
@@ -0,0 +1,104 @@
+package vochain
+
+import (
+	"fmt"
+	"math/big"
+
+	"github.com/ethereum/go-ethereum/common"
+	"go.vocdoni.io/dvote/crypto/ethereum"
+	models "go.vocdoni.io/proto/build/go/models"
+)
+
+// AddToRollingCensus adds a new key to an existing rolling census.
+// If census does not exist yet it will be created.
+func (s *State) AddToRollingCensus(pid []byte, key []byte, weight *big.Int) error {
```

Comment:
I suggest returning `fmt.Errorf("TODO")` in all unimplemented functions to avoid unexpected behaviors (like someone using this function and forgetting that it was not implemented)

Reply:
done